src/Compilers/CSharp/Test/Emit3/PartialEventsAndConstructorsTests.cs

```diff
@@ -367,19 +367,19 @@ partial C() { }
             }
             """;
         CreateCompilation(source).VerifyDiagnostics(
-            // (4,33): error CS9277: Partial member 'C.E' may not have multiple defining declarations.
+            // (4,33): error CS9402: Partial member 'C.E' may not have multiple defining declarations.
             //     partial event System.Action E;
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateDefinition, "E").WithArguments("C.E").WithLocation(4, 33),
             // (4,33): error CS0102: The type 'C' already contains a definition for 'E'
             //     partial event System.Action E;
             Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "E").WithArguments("C", "E").WithLocation(4, 33),
-            // (5,33): error CS9277: Partial member 'C.F' may not have multiple defining declarations.
+            // (5,33): error CS9402: Partial member 'C.F' may not have multiple defining declarations.
             //     partial event System.Action F;
             Diagnostic(ErrorCode.ERR_PartialMemberDuplicateDefinition, "F").WithArguments("C.F").WithLocation(5, 33),
             // (5,33): error CS0102: The type 'C' already contains a definition for 'F'
```

Comment:
>Accessibility.Public [](http://example.com/codeflow?start=25&length=20)

Is there a change in behavior around implicit accessibility in this scenario in this PR?

Reply:
Unfortunately, I cannot decipher where this comment was supposed to point to

Reply:
> Unfortunately, I cannot decipher where this comment was supposed to point to

For me, CodeFlow successfully places the comment on line 1013 in commit 6: ```Assert.Equal(Accessibility.Public, e.DeclaredAccessibility);```

Reply:
Then yes, previously the event accessors were implicitly private now they are implicitly public.